packages/core/resources/css/amazonqTransformationHub.css

```diff
@@ -0,0 +1,105 @@
+@keyframes spin {
```

Comment:
The author of this PR (https://github.com/aws/aws-toolkit-vscode/pull/4914) was unable to test it so adding his work here.